hack/make-rules/helpers/go2make/BUILD

```diff
@@ -0,0 +1,44 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
```

Comment:
I think you want to add a line
```
# gazelle:exclude testdata
```
to this BUILD file, delete all BUILD files under testdata which have been generated, and then re-run `hack/update-bazel.sh`.

you might then need to re-add the `data = glob(["testdata/**"]),` line to the `go_test` here; I'm not sure how `gazelle:exclude` affects the data attributes, if at all.

Reply:
How do you see `gazelle:exclude testdata` part being helpful?